src/lib/swapper/swappers/OneInchSwapper/utils/fetchOneInchSwap.ts

```diff
@@ -24,19 +26,11 @@ export const fetchOneInchSwap = async ({
   receiveAddress,
   sellAmountBeforeFeesCryptoBaseUnit,
   sellAsset,
-  maximumSlippageDecimalPercentage: slippage,
+  maximumSlippageDecimalPercentage,
 }: FetchOneInchSwapInput) => {
   const apiUrl = getConfig().REACT_APP_ONE_INCH_API_URL
 
-  /**
-   * limit of price slippage you are willing to accept in percentage,
-   * may be set with decimals. &slippage=0.5 means 0.5% slippage is acceptable.
-   * Low values increase chances that transaction will fail,
-   * high values increase chances of front running. Set values in the range from 0 to 50
-   */
-  const maximumSlippagePercentage = (slippage ? bnOrZero(slippage) : bnOrZero(DEFAULT_SLIPPAGE))
-    .times(100)
-    .toNumber()
+  const maximumSlippagePercentage = bnOrZero(maximumSlippageDecimalPercentage).times(100).toNumber()
```

Comment:
the comment above would be better placed here

Reply:
Right you are! Moved in https://github.com/shapeshift/web/pull/5023/commits/23d8772d13a7a5b2b1e0d0398ef04bcf2dc481e0